wallets/nami-wallet/src/index.ts

```diff
@@ -12,8 +12,8 @@ export const walletInfo: WalletInfo = {
 };
 
 /**
- * nautilus implementation of the Wallet
- * interface to be able to interact with nautilus wallet
+ * nami implementation of the Wallet
+ * interface to be able to interact with nami wallet
```

Comment:
Though these are not related to your changes, it would be nice to use upper-first version of wallet name (like the new wallets you added):
```suggestion
 * Nami implementation of the Wallet
 * interface to be able to interact with Nami wallet
```